app/test_main.py

```diff
@@ -1,3 +1,34 @@
 from app.main import get_human_age
 
-# write your code here
+
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repetition in your test cases. This will make your tests more concise and easier to maintain as per the checklist recommendation.